jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/importer/ImporterDirectoryWatcherTest.java

```diff
@@ -15,84 +15,55 @@
+import us.dot.its.jpo.ode.coder.stream.FileImporterProperties;
+import us.dot.its.jpo.ode.kafka.JsonTopics;
 import us.dot.its.jpo.ode.kafka.OdeKafkaProperties;
-import us.dot.its.jpo.ode.OdeProperties;
-import us.dot.its.jpo.ode.importer.ImporterDirectoryWatcher.ImporterFileType;
-
-public class ImporterDirectoryWatcherTest {
-
-   @Tested
-   ImporterDirectoryWatcher testImporterDirectoryWatcher;
-
-   @Injectable
-   OdeProperties injectableOdeProperties;
-   @Injectable
-   OdeKafkaProperties odeKafkaProperties;
-   @Injectable
-   Path inbox;
-   @Injectable
-   Path failureDir;
-   @Injectable
-   Path backupDir;
-   @Injectable
-   ImporterFileType injectableImporterFileType = ImporterFileType.LOG_FILE;
-   @Injectable
-   Integer timePeriod = 5;
-
-   @Capturing
-   OdeFileUtils capturingOdeFileUtils;
-   @Capturing
-   ImporterProcessor capturingImporterProcessor;
-   @Capturing
-   Executors capturingExecutors;
+import us.dot.its.jpo.ode.kafka.RawEncodedJsonTopics;
 
-   @Mocked
-   ScheduledExecutorService mockScheduledExecutorService;
-
-   // @BeforeEach
-   public void testConstructor() throws IOException {
-      new Expectations() {
-         {
-            OdeFileUtils.createDirectoryRecursively((Path) any);
-            times = 3;
-
-            Executors.newScheduledThreadPool(1);
-            result = mockScheduledExecutorService;
-         }
-      };
-   }
-
-   @Test
-   public void testRun() throws InterruptedException, IOException {
-      new Expectations() {
-         {
-            OdeFileUtils.createDirectoryRecursively((Path) any);
-            times = 3;
-
-            Executors.newScheduledThreadPool(1);
-            result = mockScheduledExecutorService;
-
-            mockScheduledExecutorService.scheduleWithFixedDelay((Runnable) any, anyLong, anyLong, TimeUnit.SECONDS);
-
-            mockScheduledExecutorService.awaitTermination(anyLong, TimeUnit.SECONDS);
-         }
-      };
-      testImporterDirectoryWatcher = new ImporterDirectoryWatcher(injectableOdeProperties, odeKafkaProperties, backupDir, failureDir, backupDir, injectableImporterFileType, timePeriod);
+import java.nio.file.Files;
+import java.nio.file.Path;
 
-      testImporterDirectoryWatcher.run();
-   }
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(initializers = ConfigDataApplicationContextInitializer.class)
+@EnableConfigurationProperties(value = {OdeKafkaProperties.class, FileImporterProperties.class, JsonTopics.class, RawEncodedJsonTopics.class})
+class ImporterDirectoryWatcherTest {
+
+    @Autowired
+    FileImporterProperties injectableFileImporterProperties;
+    @Autowired
+    OdeKafkaProperties odeKafkaProperties;
+    @Autowired
+    JsonTopics jsonTopics;
+    @Autowired
+    RawEncodedJsonTopics rawEncodedJsonTopics;
+
+
+    @Test
+    void testConstructorCreatesThreeDirectories() {
```

Comment:
note: I removed the test assertions for confirming that the scheduledExecutorService was called when `.run()` is called. I did this for a few reasons: 
1) JMockit and the Spring Configurations don't play well together (I've spent a few hours trying to make them work, but found little success)
2) The core functionality of the importerProcessor that is called by `.run()` is tested pretty fully in [LogFileToAsn1CodecPublisherTest](https://github.com/CDOT-CV/jpo-ode/pull/112/files#diff-0e3278843e1d341767a94458c97f0b0ede773febe8b9aacef568c49991bd28aa)
3) While the tests for ImporterProcessor are all disabled, no functional changes were introduced in the ImporterProcessor.
4) Once we migrate to Spring Kafka, we will need to rewrite this whole test suite anyway, specifically with regards to `.run()` since we won't be managing the threading manually anymore

I know this doesn't provide a _ton_ of confidence that the code is tested and functional, but I am comfortable with the trade-offs if the team is, too.